platform-core-module/src/main/java/com/softicar/platform/core/module/file/stored/server/AGStoredFileServer.java

```diff
@@ -13,7 +15,28 @@ public IDisplayString toDisplayWithoutId() {
 		return IDisplayString.create(getUrl());
 	}
 
-	public static Optional<AGStoredFileServer> getInstance() {
+	public boolean isPrimaryServer() {
```

Comment:
If the method below is called `getPrimary`, we should omit the tailing `Server` here as well (or add it to both method names).